lms/djangoapps/discussion_api/views.py

```diff
@@ -104,7 +104,8 @@ class ThreadViewSet(DeveloperErrorViewMixin, ViewSet):
     """
     **Use Cases**
 
-        Retrieve the list of threads for a course, retrieve thread details,
+        Retrieve the list of threads for a course (or if inline_discussion_id is provided then return
```

Comment:
No need to add "discussion_id" in the docstring here, since this docstring doesn't specify all the query params.  Rather, the comprehensive list of filter params are documented below.